OnePiece_d20/sheet.css

```diff
@@ -0,0 +1,206 @@
+.sheet-main {
+  display: grid;
+  width: 100%;
+  height: auto;
+  background-color: lightgray;
+  /*grid-gap: 4px;*/
+  grid-template-columns: 25% 25% 25% 25%;
+  grid-template-rows: 100px 450px 300px 1fr 2fr 2fr;
```

Comment:
Sheet submission looking good all around.
If you want the sheet to be a bit more compact, you could experiment with different heights on the `450px` & `300px` if you think there is too much space between rows, at least based on the preview there is plenty of space between the rows.

Out of curiosity, did you use the Roll20-grid-template or the Cyberpunk Red sheet as a base/inspiration? The code looked imminently familiar, and reminded me that I haven't updated my original sheet-template at all, even if I've evolved somewhat on what I put on my later sheets.

Reply:
The Roll20-grid-template as a base, then expanded as needed.

Good point on the specific row heights, could definitely mess with that to compact it some